### ISSUE TITLE
Feat: Quick Links in the footer are not working

### ISSUE THREAD

**Issue by meghana9044** [Its-Aman-Yadav/Community-Site#262]
### Describe the feature

I would like to solve this issue. In the quick links,
1. I will link the About Us to About page
2. I will link the Contact Us to Get In touch
3. I will link the Support to FAQ

### Add ScreenShots

<img width="1440" alt="Screen Shot 2024-05-16 at 10 12 22 PM" src="https://github.com/Its-Aman-Yadav/Community-Site/assets/105973009/8d052b58-eaa1-42a9-9932-ecc3b56af746">

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue

**Comment by PavanKumarNaik936** [comment]
yes, the links which are in the footer section are not working even the social media icons also not navigating to their respective sites. @Its-Aman-Yadav so, can you assign this issue to me i will resolve this and i am a gssoc '24 contributor
thankyou.

**Comment by Its-Aman-Yadav** [comment]
Sure @meghana9044

**Comment by SHANMUGAPRIYA-1** [comment]
Please do assign this task to me

**Comment by shikha2319** [comment]
Please, Assign this task to me.
